docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Messaging.adoc

```diff
@@ -639,6 +639,30 @@ This is done from the CLI:
     }
 }
 ----
+
+=== Critical analyzis of the broker
+
+When things go wrong on the broker, the critical analyzer acts as a safeguard shtting down the broker. +
```

Comment:
From my understanding, shutdown is neither default behavior or only option...

Reply:
also s/shtting/shutting

Reply:
Made it clearer